gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -49,10 +53,22 @@ const NxPageTitlePage = () =>
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-page-title__page-icon</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-page-title__headings</code></td>
+            <td className="nx-cell"><code className="nx-code"></code></td>
```

Comment:
The empty `.nx-code` doesn't look right.  Either the whole cell should be empty, or ideally this should have a convenience component

Reply:
> The empty `.nx-code` doesn't look right. Either the whole cell should be empty, or ideally this should have a convenience component

That was a mistake: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/6ceb255ac0af3789c1fc381f299d11a089419a36